### ISSUE TITLE
Support for generating the managing API tokens

### ISSUE THREAD

**Issue by deepshikha-jha** [Cogniac/cogniac-sdk-csharp#13]

**Comment by deepshikha-jha** [comment]
https://github.com/Cogniac/cogniac-sdk-py/commit/30d8c07d46754c73aee526c41e5ba1c9ecd6c436

**Comment by alkadi** [comment]
Ok, I will mimicked the Python code, but I can't find the API responses and URLs in your documentation (for class design) at https://cogniac.readme.io/docs/users

**Comment by wskish** [comment]
@deepshikha-jha can you please update the readme docs for these new endpoints?

**Comment by deepshikha-jha** [comment]
Yeah, we have more updates to the API documentation as well. Coming up soon

**Comment by deepshikha-jha** [comment]
Sure, I will update the API documentation with these

**Comment by alkadi** [comment]
Thank you @deepshikha-jha, please ping me when they are updated.

**Comment by deepshikha-jha** [comment]
https://cogniac.readme.io/docs/users-api-key-management
